numpy/random/dsfmt.pyx

```diff
@@ -128,8 +127,8 @@ cdef class DSFMT:
 
     >>> from numpy.random.entropy import random_entropy
     >>> from numpy.random import Generator, DSFMT
-    >>> seed = random_entropy()
-    >>> bit_generator = DSFMT(seed)
+    >>> seed_seq = SeedSequence()
+    >>> bit_generator = DSFMT(seed_seq)
```

Comment:
This goes in my "evidence that `jumped()` is easily misused" file. :-)

Reply:
It was just a documentation error -- the jump used to be in place, and all that was done was to change jump->jumped.

Reply:
I am confused. Why does changing the order of operations matter?

Reply:
Actually, I think the example below is more correct and safe.

```
    >>> bit_generator = DSFMT(seed_seq)
    >>> rg = []
    >>> for _ in range(10):
    ...    rg.append(Generator(bit_generator))
    ...    # Chain the BitGenerators
    ...    bit_generator = bit_generator.jumped()
```
This creates a list of 10 `Generator`s with their own `BitGenerator`s that you can send to 10 processes and leaves you one left-over `BitGenerator` to use in the remainder of this code and maybe create some more.

The alternate code leaves `bit_generator` pointing to the last `BitGenerator` that was added to the list, which means that it could be used for more jumping, but shouldn't be used to generate any numbers in the present process.

This is all very finicky. What I fear that people will do the more obvious, more convenient, but less safe

```python
bit_generator = DSFMT(seed_seq)
rg = [Generator(bit_generator.jumped(i)) for i in range(1, 11)]
```

Which is generally fine by itself, but if they ever need to create more PRNGs, they will need to track how many they've created themselves to prevent overlap.

Reply:
Now that dsfmt has been removed, this is not really relevant. I will add it to numpy/bitgenerators/dsfmt.pyx